config/overlays/development/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+bases:
```

Comment:
I'm not clear on the "overlays" convention. Can you explain what this means?

Reply:
I'm following [this](https://github.com/kubernetes-sigs/kustomize#2-create-variants-using-overlays) but happy to defer to more experienced Kustomize users.

Reply:
Makes complete sense. Thanks!